src/main/java/org/spongepowered/common/service/permission/SpongePermissionService.java

```diff
@@ -180,4 +185,13 @@ public void addDescription(PermissionDescription permissionDescription) {
     public SpongeSubjectCollection getDefaultCollection() {
         return defaultCollection;
     }
+
+    public void setupDefaultPermissions() {
```

Comment:
If it's being used, it should be javadoc'ed as such to avoid future confusion (I've had other methods that were made public in the past and when I tried to rename them or change them, they'd magically fail on either implementation because of the lack of documentation ;))